README.md

```diff
@@ -37,15 +40,17 @@ language, so that the l10n coordinator only needs to interact with one
 person per language.
 
 
-Creating a new language translation (direct translation)
+## Creating a new language translation (direct translation)
+
 --------------------------------------------------------
 
 Alternatively, you can start translating in your own language by
 registering to Weblate and starting a new language translation at
 
 https://hosted.weblate.org/projects/git-manpages/translations/
 
-Translation Process Flow
+## Translation Process Flow
+
 ------------------------
 
 The overall data-flow looks like this:
```

Comment:
The ASCII art that follows will probably get mangled by markdown rendering. Might need to be wrapped in a code block or converted to an svg that can be embedded.

Reply:
The parser recognizes it as a code block due to the indentation.